doc/sphinx/proofs/automatic-tactics/logic.rst

```diff
@@ -153,6 +153,7 @@ Solvers for logic and equality
    equalities in the context. Otherwise it tries to infer a discriminable equality
    from those in the context. Alternatively, congruence tries to prove that a
    hypothesis is equal to the goal or to the negation of another hypothesis.
+   Written by Pierre Corbineau.
```

Comment:
Looks like a bad rebase artefact.

Reply:
No, intentional.  As we discussed recently, attributions should be in the changes log, not the main text.

Reply:
What was intentional was the removal in the previous PR, not the reintroduction in this one, I suppose.

Reply:
Got it.  Eventually, after sufficient repetition, I am usually able to get things right ;-)